<?xml version="1.1" encoding="UTF-8"?>
<launch>

    <node pkg="dynamic_tracker" name="cluster_test_node" type="cluster_test_node" output="screen">
        <param name="cluster_type" value="2"/>
        <param name="epsilon" value="0.5"/>
        <param name="min_points" value="5"/>
        <param name="odom_topic" value="/iris/mavros/local_position/odom"/>
        <param name="cloud_topic" value="/cloud_registered"/>
        <param name="use_adaptive_dbscan" value="true"/>
        <param name="sensing_rate" value="10.0"/>
        <param name="frame_id" value="camera_init"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find dynamic_tracker)/config/default.rviz" />

    <node pkg="rosbag" type="play" name="player"  args="--clock /home/huyu/bags/gazebo2long.bag"/>

</launch> 